### PR TITLE
Add nightly VTK master CI

### PR DESCRIPTION
**Pull request by mwestphal**
- Run a subset of CI nightly against the master branch of VTK

### CODE REVIEW

**Comment by mwestphal** [comment]
Completed nightly CI can be seen here: https://github.com/f3d-app/f3d/actions/runs/6020907706

File: .github/workflows/nightly_vtk_master.yml [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    - cron:  '0 0 * * *'
+
+jobs:
+
+#----------------------------------------------------------------------------
+# Check nightly: Checkout VTK and check if the last merged commit is less than 24 hours old
+#----------------------------------------------------------------------------
+  check_nightly:
+    runs-on: ubuntu-latest
+    name: Check nightly is needed
```
Comment (Meakk) [inline .github/workflows/nightly_vtk_master.yml:14]:
```suggestion
    name: Check nightly if needed
```

**Review by Meakk — APPROVED** [review]
typos then good to go

File: .github/workflows/nightly_vtk_master.yml (hunk 1 above)
Comment (mwestphal) [reply .github/workflows/nightly_vtk_master.yml:14]: hum no, this action "check if the nightly is needed". I'll just rename it "Check nightly".

**Comment by mwestphal** [comment]
Complete green run: https://github.com/f3d-app/f3d/actions/runs/6033937509

**Pull request merged**